src/main/java/org/broadinstitute/hellbender/tools/funcotator/BaseFuncotatorArgumentCollection.java

```diff
@@ -22,9 +23,11 @@
      * This variable is necessary to resolve the differences between b37 and hg19 when
      * dealing with Homo Sapiens samples.
      */
-    protected static String FuncotatorReferenceVersionHg19 = "hg19";
+    @VisibleForTesting
+    public static String FuncotatorReferenceVersionHg19 = "hg19";
     /** String representing the hg38 version of the homo sapiens reference. */
-    protected static String FuncotatorReferenceVersionHg38 = "hg38";
+    @VisibleForTesting
+    public static String FuncotatorReferenceVersionHg38 = "hg38";
```

Comment:
These can't be final? Also, generally when something is `@VisibleForTesting`, the test class is in the same package and so default/package access is sufficient rather than public access.

Reply:
Oops.  They definitely should be final.  Should have done that when I originally implemented this.  

Unfortunately they cannot be package-private.  `DbSnpIntegrationTest` is not in the same package as `BaseFuncotatorArgumentCollection` so needs the extra visibility.